tests/deprecated/test_repo.py

```diff
@@ -15,7 +15,7 @@
 
 
 @patch("appdirs.user_cache_dir", return_value="root")
-@patch("moban.utils.mkdir_p")
+@patch("moban.core.utils.mkdir_p")
```

Comment:
This bug was hidden for a while now.

Reply:
It is strange that nose tests did not fail.

Reply:
Thanks! There are more such places, I do not know about pytest/nose enough to tell why it have not manifest earlier.